core/src/main/scala/kafka/coordinator/transaction/ProducerIdManager.scala

```diff
@@ -167,7 +167,7 @@ class RPCProducerIdManager(brokerId: Int,
       if (nextProducerId > currentProducerIdBlock.lastProducerId) {
         val block = nextProducerIdBlock.poll(maxWaitMs, TimeUnit.MILLISECONDS)
         if (block == null) {
-          throw Errors.REQUEST_TIMED_OUT.exception("Timed out waiting for next producer ID block")
+          throw Errors.COORDINATOR_LOAD_IN_PROGRESS.exception("Timed out waiting for next producer ID block")
```

Comment:
It would be helpful to have a comment here since it's probably not obvious why we use this error instead of `REQUEST_TIMED_OUT`.